NewForce-Cohort-4/tabloid-full-stack-team-fuschia
# Dtb initial setup

# Description
Fixed the authorization bugs and finished Ticket 4 - My Posts.

## Type of change
Please delete options that are not relevant.

# Testing Instructions
- Start the Web API and React App
- Login to the App. On page load, you should see the word " Hello" 
- Select the My Posts link in the Menu Navigation at the top of the page
- Ensure that each of the listed posts was originally posted by the current user (you)
- Click "Logout" and make sure you are redirected to the Login page.

# Checklist:
